docs/topics/Gateway_Events.md

```diff
@@ -1161,6 +1162,31 @@ Sent when properties about the current bot's user change. Inner payload is a [us
 
 ### Voice
 
+#### Voice Channel Effect Send
+
+Sent when someone sends an effect, such as an emoji reaction or a soundboard sound, in a voice channel the current user is connected to.
+
+###### Voice Channel Effect Send Event Fields
+
+| Field                | Type                                                | Description                                                                                                                                     |
+| -------------------- | --------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------- |
+| channel_id           | snowflake                                           | ID of the channel the effect was sent in                                                                                                        |
+| guild_id             | snowflake                                           | ID of the guild the effect was sent in                                                                                                          |
+| user_id              | snowflake                                           | ID of the user who sent the effect                                                                                                              |
+| emoji?               | ?[emoji](#DOCS_RESOURCES_EMOJI/emoji-object) object | The emoji sent, for emoji reaction and soundboard effects                                                                                       |
+| animation_type?      | integer                                             | The [type of emoji animation](#DOCS_TOPICS_GATEWAY_EVENTS/voice-channel-effect-send-animation-types), for emoji reaction and soundboard effects |
+| animation_id?        | integer                                             | The ID of the emoji animation, for emoji reaction and soundboard effects                                                                        |
+| sound_id?            | snowflake                                           | The ID of the soundboard sound, for soundboard effects                                                                                          |
```

Comment:
Does not necessarily have to be a snowflake. It's a number when it's a default sound.

Reply:
looks like for default sounds the id is in a string (e.g. `"1"`) in the API but a number in this gw event 🫠

Reply:
> looks like for default sounds the id is in a string (e.g. `"1"`) in the API but a number in this gw event 🫠
> 
> updated to "snowflake or integer"

@advaith1 has this been fixed?

Reply:
no, this is staying as-is for now. you should treat `1` and `"1"` as identical in payloads of this event - do not make assumptions based on whether it is a string or not.